sdk/go/common/tokens/names.go

```diff
@@ -94,7 +94,7 @@ func IntoQName(s string) QName {
 
 // AsQName converts a given string to a QName, asserting its validity.
 func AsQName(s string) QName {
-	contract.Assertf(IsQName(s), "Expected string '%v' to be a name (%v)", s, QNameRegexpPattern)
+	contract.Assertf(IsQName(s), "Expected string '%v' to be a qname (%v)", s, QNameRegexpPattern)
```

Comment:
In case this can ever surface to the user, expand that a bit, what's a qname? "Expected name '%v' to match $QNameRegexpPattern regular expression" or something like this?

Reply:
This is an assert, so we don't really expect it to be surfaced to users unless they were using this in their own code. At which point they've called "AsQName" so they must know what a qname is.

Reply:
Ah yeah it's fine then.